x-pack/plugins/security_solution/public/onboarding/components/onboarding_header/onboarding_header.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  COLOR_MODES_STANDARD,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import { useCurrentUser } from '../../../common/lib/kibana/hooks';
+import { useOnboardingHeaderStyles } from './onboarding_header.styles';
+import rocketImage from './images/header_rocket.png';
+import rocketDarkImage from './images/header_rocket_dark.png';
+import { TeammatesCard } from './cards/teammates_card';
+import { VideoCard } from './cards/video_card';
+import { DemoCard } from './cards/demo_card';
+import * as i18n from './translations';
+
+export const OnboardingHeader = React.memo(() => {
+  const currentUser = useCurrentUser();
+  const { colorMode } = useEuiTheme();
+  const isDarkMode = colorMode === COLOR_MODES_STANDARD.dark;
+
+  const styles = useOnboardingHeaderStyles();
+
+  // Full name could be null, user name should always exist
+  const currentUserName = currentUser?.fullName || currentUser?.username;
+
+  return (
+    <>
+      <EuiFlexGroup justifyContent="center" alignItems="center" className={styles}>
+        <EuiFlexItem grow={false}>
+          <EuiImage
+            src={isDarkMode ? rocketDarkImage : rocketImage}
+            size={128}
+            alt={i18n.GET_STARTED_DATA_INGESTION_HUB_SUBTITLE}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false} className="onboardingHeaderTitleWrapper">
+          {currentUserName && (
+            <EuiTitle size="xs" className="onboardingHeaderGreetings">
+              <span>{i18n.GET_STARTED_PAGE_TITLE(currentUserName)}</span>
```

Comment:
just double checking: currentUserName can be undefined, but in the translation we just pass the undefined into translation, which might be Hi, undefined. Because stringified `undefined` is `"undefined"`, unless of course react intl does something about it.

Reply:
There's the `currentUserName &&` just above. If it's `undefined` the translation is not called

Reply:
oh my bad missed that